src/Console/Application.php

```diff
@@ -34,13 +40,60 @@ class Application extends BaseApplication
      */
     private $container;
 
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    /**
+     * @var bool
+     */
+    private $isXdebugLoaded;
+
+    /**
+     * @var bool
+     */
+    private $isDebuggerDisabled;
+
     public function __construct(Container $container, string $name = self::NAME, string $version = self::VERSION)
     {
         $this->container = $container;
+        $this->isDebuggerDisabled = empty((string) getenv(XdebugHandler::ENV_DISABLE_XDEBUG));
```

Comment:
`'' !== trim((string) getenv(XdebugHandler::ENV_DISABLE_XDEBUG))`; Not strictly necessary but personally I prefer to avoid any usage of `empty()` which is gun footing